Assets/Scripts/Player/AimMode.cs

```diff
@@ -144,16 +147,33 @@ public void ThrowObject()
 
     public void changeCameraToPossession()
     {
-        ExitAimMode();
         if (_possessionController.CurrentPossession == null)
         {
+            ExitAimMode();
             return;
         }
         Transform pos = _possessionController.CurrentPossession.GetComponent<ClutterCamera>().CinemachineCameraTarget.transform;
-        _possessionDefaultCam.LookAt = pos;
-        _possessionDefaultCam.Follow = pos;
-        _possessionAimCam.LookAt = pos;
-        _possessionAimCam.Follow = pos;
+        CinemachineVirtualCamera newDefaultCamera;
+        CinemachineVirtualCamera newAimCamera;
+        if (_possessionDefaultCam == _cameras[2])
+        {
+            newDefaultCamera = _possessionDefaultCam1;
+            newAimCamera = _possessionAimCam1;
+
+        } else
+        {
+            newDefaultCamera = _possessionDefaultCam;
+            newAimCamera = _possessionAimCam;
+        }
```

Comment:
This is difficult to read without context. This operation could be described in a short sentence SwapPossessionCamera or something along those lines. If you put this in a function with a name like that and call it it makes the code that much more readable.